bin/bench

```diff
@@ -3,16 +3,32 @@
 const path = require('path');
 const program = require('commander');
 const run = require('../lib/run');
+const mkdirp = require('mkdirp').sync;
+const { dirname, readFileSync, writeFileSync } = require('fs');
 
 let input = path.resolve('bench');
 
 program
   .version('0.1.0')
   .arguments('<directory>')
   .option('-i, --iterations <n>', 'The number of iterations to run', parseInt)
-  .action((directory) => input = path.resolve(directory))
-  .parse(process.argv);
+  .action((directory) => {
+    let iterations = program.iterations || 1;
+    directory = path.resolve(directory) || input;
```

Comment:
`path.resolve()` will always return a value or throw on non-string values, so we'll never get the default behavior here.